### ISSUE TITLE
Minimally invasive UI option

### ISSUE THREAD

**Issue by roedoejet** [ReadAlongs/Web-Component#2]
There should be a very minimal, stripped down version (no padding/margins/borders) for folks that want to integrate into existing sites like http://ynlc.ca/dakeyi/03hutshi/04chemia/chemia.html

**Comment by joanise** [comment]
Seems moot to me, no? I'm pretty sure the current web-component meets that requirement.

**Comment by roedoejet** [comment]
No, it doesn't really - Chris Cox wanted something that could be embedded on an arbitrary div that already exists on a webpage. I'm not sure we will be able to do that, it's somewhat of a separate project.
